ngmt/modules/ssd/_pham.py

```diff
@@ -186,6 +200,9 @@ def detect(
         stationary_2 = accel_var <= threshold
         stationary_2 = (accel_var <= threshold).astype(int)
```

Comment:
Why do you calculate `stationary_2` twice? Can we delete one line?

Reply:
I handled this issue by eliminating the redundant line of the code.